engine/src/main/java/org/terasology/engine/config/flexible/ui/AutoConfigScreen.java

```diff
@@ -46,7 +46,7 @@ public void initialise() {
             if (widget.isPresent()) {
                 mainContainer.addWidget(widget.get());
             } else {
-                logger.warn("Cannot create widget for config: {}", config.getId());
+                logger.atWarn().addArgument(() -> config.getId()).log("Cannot create widget for config: {}");
```

Comment:
Simple getter method so impact neglectable
```suggestion
                logger.warn("Cannot create widget for config: {}", config.getId()); //NOPMD
```